rpcclient/api.go

```diff
@@ -246,11 +246,11 @@ func (c *Client) TestResults(pid []byte, totalVotes int, withWeight uint64) (res
 		}
 		log.Infof("no results yet")
 	}
-	total := fmt.Sprintf("%d", uint64(totalVotes)*withWeight)
-	if results[0][1] != total ||
-		results[1][2] != total ||
-		results[2][3] != total ||
-		results[3][4] != total {
+	total := uint64(totalVotes) * withWeight
```

Comment:
You can make total a big.Int here so you don't need to declare new variables on each comparison.

Reply:
yup! changed